R/get_manta_sv.R

```diff
@@ -0,0 +1,170 @@
+#' or the complete region specified under `region` (in chr:start-end format), note that chromosome can be either prefixed or not prefixed.
+#' Useful filtering parameters are also available, use `min_vaf` to set the minimum tumour VAF for a SV to be returned and `min_score`
+#' to set the lowest Manta somatic score for a SV to be returned. `pair_status` can be used to return variants from either matched or unmatched samples.
+#' In addition, the user can chose to return all variants, even the ones not passing the filter criteria. To do so, set `pass = FALSE` (default is TRUE).
+#' Warning, the following parameters are not available in this version of get_manta_sv; `from_flatfile`, `from_cache`, `write_to_file`.
+#' These parameters are only used in [GAMBLR.results::get_manta_sv]. 
+#'
+#' @param these_sample_ids A vector of multiple sample_id (or a single sample ID as a string) that you want results for.
+#' @param these_samples_metadata A metadata table to auto-subset the data to samples in that table before returning.
+#' @param projection The projection genome build. Default is grch37.
+#' @param chromosome Optional, the chromosome you are restricting to (can be prefixed or not prefixed).
+#' @param qstart Optional, query start coordinate of the range you are restricting to.
+#' @param qend Optional, query end coordinate of the range you are restricting to.
+#' @param region Optional, region formatted like chrX:1234-5678 (chromosome can be prefixed or not prefixed) instead of specifying chromosome, start and end separately.
+#' @param pairing_status Use to restrict results (if desired) to matched or unmatched results (default is to return all). This parameter takes the filtering condition as a string ("matched" or "unmatched").
+#' @param min_vaf The minimum tumour VAF for a SV to be returned. Default is 0.1.
+#' @param min_score The lowest Manta somatic score for a SV to be returned. Default is 40.
+#' @param pass If TRUE (default) only return SVs that are annotated with PASS in the FILTER column. Set to FALSE to keep all variants, regardless if they PASS the filters.
+#' @param verbose Set to FALSE to minimize the output to console. Default is TRUE. This parameter also dictates the verbose-ness of any helper function internally called inside the main function.
+#' @param from_cache This parameter does not do anything for this version of get_manta_sv. See [GAMBLR.results::get_manta_sv] for more info.
+#' @param from_flatfile This parameter does not do anything for this version of get_manta_sv. See [GAMBLR.results::get_manta_sv] for more info.
+#' @param write_to_file This parameter does not do anything for this version of get_manta_sv. See [GAMBLR.results::get_manta_sv] for more info.
+#' 
+#' @export
+#' 
+#' @examples
+#' #lazily get every SV in the table with default quality filters
+#' all_sv = get_manta_sv()
+#'
+#' #get all SVs for a single sample
+#' some_sv = get_manta_sv(these_sample_ids = "DOHH-2")
```

Comment:
Can you extend this example to a more "real world" use case? You could use `get_gambl_metadata` to get all samples in a specific cohort (e.g. DLBCL_cell_lines) then provide the metadata table instead of `these_sample_ids`.

Reply:
For sure, I'll add it in my next round of commits.